packages/react/src/components/Alert/Alert.tsx

```diff
@@ -137,8 +137,5 @@ class Alert extends UIComponent<WithAsProp<AlertProps>, AlertState> {
 /**
  * A Alert displays information that explains nearby content.
  * @accessibility
```

Comment:
should we remove @accessibility section altogether, as there is no content there?

Reply:
I think this would be a valid update.